### PR TITLE
ISSUE-65: Train a custom model

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This pull request contains a Ruuter Endpoint which forwards the request to the Machine Learning service responsible for training models.

### CODE REVIEW

File: DSL.Ruuter/GET/training_status.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+get_status:
+  call: http.get
+  args:
+    url: http://ml-service:8000/status
```
Comment (turnerrainer) [inline DSL.Ruuter/GET/training_status.yml:4]: What endpoint is `http://ml-service:8000/status`?

File: DSL.Ruuter/POST/train.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+train_model:
+  call: http.post
+  args:
+    url: http://ml-service:8000/task
```
Comment (turnerrainer) [inline DSL.Ruuter/POST/train.yml:4]: What endpoint is `http://ml-service:8000/task`?

File: README.md [hunk 3]
```diff
@@ -13,3 +13,18 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
 ```
+
+### Train a custom model
+
+https://github.com/buerokratt/Data-Anonymizer/issues/65
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/train.yml
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 4]
```diff
@@ -13,3 +13,18 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/GET/training_status.yml
+
+```
+```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: DSL.Ruuter/GET/training_status.yml (hunk 1 above)
Comment (alimuhammadahmer) [reply DSL.Ruuter/GET/training_status.yml:4]: This is the endpoint which is hosted on our ML Service responsible for training a model. It returns the current status of training.

File: DSL.Ruuter/POST/train.yml (hunk 2 above)
Comment (alimuhammadahmer) [reply DSL.Ruuter/POST/train.yml:4]: This is the endpoint which is hosted on our ML Service responsible for training a model. It initiates the training of a model based on the corpora.

File: README.md (hunk 3 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (alimuhammadahmer) [reply README.md]: Done

**Pull request merged**